File: colorPickerView/src/main/java/com/hzy/myapplication2/MainActivity.java
package com.hzy.myapplication2;

import androidx.appcompat.app.AppCompatActivity;
import android.os.Bundle;
import android.widget.FrameLayout;

public class MainActivity extends AppCompatActivity {
    ProgressView progressView;
    FrameLayout container;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        container = findViewById(R.id.container);
        FormXCJCBean titleBean2 = new FormXCJCBean();
        titleBean2.total = 10;
        titleBean2.yzg = 5;
        titleBean2.dfy = 2;
        titleBean2.dzg = 0;
        titleBean2.fzcgb = 3;
        progressView =  new ProgressView(this,titleBean2);
        container.addView(progressView);
    }
}

File: baseKotlinFrame/src/main/java/com/example/basekotlinframe/MainActivity.kt
package com.example.basekotlinframe

import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import android.widget.Toast
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        tvTitle.setText("dsfsdfsdfs")
        tvTitle.setOnClickListener{
            Toast.makeText(this,"1111",Toast.LENGTH_SHORT).show()
        }
    }
}
